changelog/59153.fixed

```diff
@@ -0,0 +1 @@
+Fix issue where passed port was being ignored
```

Comment:
Can you add more detail to the changelog entry, so users will know this impacts salt-cloud and windows?

Reply:
Dun